hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/ObjectStore.java

```diff
@@ -551,14 +551,81 @@ public void deleteSnapshot(String volumeName,
 
   /**
    * List snapshots in a volume/bucket.
-   * @param volumeName volume name
-   * @param bucketName bucket name
-   * @return list of snapshots for volume/bucket snapshotpath.
+   * @param volumeName     volume name
+   * @param bucketName     bucket name
+   * @param snapshotPrefix snapshot prefix to match
+   * @param prevSnapshot   snapshots will be listed after this snapshot name
```

Comment:
is prev snapshot snapshot chain order based on the snapshot creation time or snapshot name lexicographical order ?

Reply:
They are based on lexicographical order.